docker/Dockerfile.base

```
@@ -9,6 +9,8 @@
 # Base image
 ARG ISAACSIM_BASE_IMAGE_ARG
 ARG ISAACSIM_VERSION_ARG
+ARG CONTROL_LIBRARIES_TAG=v8.1.0
```

```suggestion
ARG CONTROL_LIBRARIES_TAG=v9.2.0-jammy
```
should hopefully work now